python/Ganga/GPIDev/Lib/Tasks/ITransform.py

```diff
@@ -128,6 +128,9 @@ def __init__(self):
         super(ITransform, self).__init__()
         self.initialize()
 
+    def _auto__init__(self):
+        self.status = 'new'
```

Comment:
Is this needed even though the schema should set the default value? I worry that something is broken if this *is* needed.